source/lex.tex

```diff
@@ -39,19 +39,8 @@
 called a \defnadj{preprocessing}{translation unit}.
 \begin{note}
 A \Cpp{} program need not all be translated at the same time.
-\end{note}
-
-\pnum
-\begin{note}
-Previously translated translation units and instantiation
-units can be preserved individually or in libraries. The separate
-translation units of a program communicate\iref{basic.link} by (for
-example)
-calls to functions whose identifiers have external or module linkage,
-manipulation of objects whose identifiers have external or module linkage, or
-manipulation of data files. Translation units can be separately
-translated and then later linked to produce an executable
-program\iref{basic.link}.
+Translation units can be separately translated and then later linked to
+produce an executable program\iref{basic.link}.
```

Comment:
```suggestion
Translation units can be separately translated and then linked together to
produce a program\iref{basic.link}.
```
This change would bring the note more in line with the choice of words in https://eel.is/c++draft/basic.link#1

Reply:
For now I was deliberately restricting myself to moving approved words around.

I have no objections to further refining the existing text, but would rather do that as a follow-up once the text is in its new place.

Reply:
For the record, my own rewording would be more like:
"Source files can be separately translated to produce translation units and instantiation units that are later linked to produce an executable program."

Reply:
side note: Of course, "instantiation units" no longer exist, really, given that we (must) have interleaved instantiation and constant evaluation.

Reply:
Are we still OK with this move then?

Reply:
Or should we wait for Reflection to land first?

Reply:
I think this would have made a good addition for N5001, so that the reflection folks could have had this slightly cleaner base to work from.  As it missed P5001, I suggest we refer the authors of P2996 to look at this and pick up anything they find useful.